### ISSUE TITLE
The min-elements value cannot be greater than the max-elements value in leaf-list/list statements

### ISSUE THREAD

**Issue by fredgan** [mbj4668/pyang#564]
Hi,
I find that it can't report any errors when the `min_elements` value is greater than the `max_elements` value in leaf-list or list statements. The YANG module is as follows:

the version is 2.1.1.

`mod1.yang:`
```
module mod1 {
  yang-version 1.1;
  namespace "urn:mod1";
  prefix m1;

  leaf-list le1 {
    type string;
    max-elements 50;
    min-elements 51; // error: bad value
  }

  list lst {
    key name;
    leaf name {
      type string;
    }
    max-elements 50;
    min-elements 51; // error: bad value
  }
}
```
`the expected result:`
```
$ pyang -f yang mod1.yang
mod1.yang:9: error: a 'min-elements' value cannot be greater than  'max-elements' value
mod1.yang:18: error: a 'min-elements' value cannot be greater than  'max-elements' value
module mod1 {
  yang-version 1.1;
  namespace "urn:mod1";
  prefix m1;

  leaf-list le1 {
    type string;
    max-elements 50;
    min-elements 51; // error: bad value
  }
  list lst {
    key "name";
    leaf name {
      type string;
    }
    max-elements 50;
    min-elements 51; // error: bad value
  }
}
```
but the actual result is no errors.

**Comment by JieZhang0822** [comment]
Hi fredgan,

I used the following yang file to test 2 cases:

pyang mod1.yang --print-error-code

module mod1 {
  yang-version 1.1;
  prefix bb;
  namespace "urn:mod1";

  leaf-list le1 {
    type string;
    max-elements "ab";// error: bad value
    min-elements 50;
  }

  list lst {
    key "name";
    leaf name {
      type string;
    }

    max-elements 50;
    min-elements "cd";// error: bad value
  }
}

**Case 1: no bugfix with "#564"**
The output is :
mod1.yang:8: error: BAD_VALUE
mod1.yang:19: error: BAD_VALUE

**Case 2: with bugfix with "#564"** 
The output is:
    if int(min_value.arg) > int(max_value.arg):
ValueError: invalid literal for int() with base 10: 'ab'

The reason is that the fix of #564 doesn't handle the value error for min-elements and max-elements.

Thanks

Jie

**Comment by fredgan** [comment]
@JieZhang0822 Thanks Jie. Fixed now with the new PR
